### PR TITLE
Part of #17670: Replace Typography with Text component in beta-header/index.js

### PR DESCRIPTION
**Pull request by dhruvv173**
## Explanation
This PR partially fixes [#17670](https://github.com/MetaMask/metamask-extension/issues/17670)
made changes to the ```beta-header/index.js``` component
ui/components/app/beta-header/index.js

### Before
image

### After
image

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Comment by dhruvv173** [comment]
Hey @georgewrmarshall @NidhiKJha
Could you guys please review this PR? It finally passed all the tests, attached the before/after screenshot and updated the snapshot as well.
Thanks!

File: ui/components/app/beta-header/index.js [hunk 1]
```diff
@@ -28,8 +32,9 @@ const BetaHeader = () => {
       className="beta-header"
       alignItems={AlignItems.center}
     >
-      <Typography
-        variant={TypographyVariant.H7}
+      <Text
+        variant={TextVariant.bodySm}
+        as="h6"
         marginTop={0}
         marginBottom={0}
```
Comment (georgewrmarshall) [inline ui/components/app/beta-header/index.js:39]: The Text component doesn't have any default margins so you could remove these
```suggestion
```

**Review by georgewrmarshall — COMMENTED** [review]
Looks great left one suggestion on an improvement that I missed in the other PR

File: ui/components/app/beta-header/index.js (hunk 1 above)
Comment (dhruvv173) [reply ui/components/app/beta-header/index.js]: I have made the suggested changes.
Please review the commit, thanks!

File: ui/components/app/beta-header/index.js [hunk 2]
```diff
@@ -28,10 +32,9 @@ const BetaHeader = () => {
...
     >
-      <Typography
-        variant={TypographyVariant.H7}
-        marginTop={0}
-        marginBottom={0}
+      <Text
+        variant={TextVariant.bodySm}
+        as="h6"
         className="beta-header__message"
         color={Color.warningInverse}
```
Comment (DDDDDanica) [inline ui/components/app/beta-header/index.js:39]:
```suggestion
        color={TextColor.warningInverse}
```
You can use this enum to define text color :)

Comment (dhruvv173) [reply ui/components/app/beta-header/index.js]: I have made the changes in the code. Could you please review it?
Thanks!

**Review by DDDDDanica — APPROVED** [review]

**Review by NidhiKJha — APPROVED** [review]
LGTM! Thanks for your contribution @dhruvv173

**Pull request merged**